src/actions/historyActions.js

```diff
@@ -71,7 +71,7 @@ import type SDKWrapper from 'services/api';
 import type { Dispatch, GetState } from 'reducers/rootReducer';
 
 // actions
-import { fetchAssetsBalancesAction, loadSupportedAssetsAction } from './assetsActions';
+import { fetchAssetsBalancesAction } from './assetsActions';
```

Comment:
Maybe it would be good idea to have separate files for archanova & etherspot, eg. archanovaHistoryActions.js & etherspotHistoryActions.js. From my perspective it seems that the data format is quite different, so it's better to keep them separated to improve readability. Wdyt @poocart ?

Reply:
imho leaving `historyActions` is still a good way to keep things organized as if we split history actions as account-service-oriented then we would increase repeated reducer updates for the same history reducer

however, I'll take a look into this once I start working with history 👍